### DISCUSSION TITLE
Use `gg` but exclude other `g<keys>`?

### DISCUSSION THREAD

**Discussion by xfzv** [gdh1995/vimium-c#1120] [Q&A] [answered]
On a specific website, some `g+<keys>` shortcuts are already defined (but `gg` is not).

I'd like to use `gg` from Vimium C while still being able to use other said `g+<keys>` shortcuts from the website.

> if "Keys" starts with "^", only listed keys are enabled,
and all others are passed through;
otherwise, just the listed keys are passed through.

With `^ g g` rule, `gg` works as expected but the `g+<keys>` website shortcuts don't (Vimium C ones are used instead).

Is there a workaround?

Thanks.

**Comment by gdh1995** [comment, marked as answer]
No there's no such a way.

But you may bind another key to `scrollToTop`, like:
```
map Gg scrollToTop
```

**Reply by xfzv** [reply]
I see. Thank you for the answer and the suggestion!
